src/users/users.controller.ts

```diff
@@ -1,25 +1,34 @@
 import { Controller, Get, Param, Request, UseGuards } from "@nestjs/common";
 import { UsersService } from "./users.service";
-import { ApiBearerAuth, ApiTags } from "@nestjs/swagger";
+import { ApiBearerAuth, ApiOperation, ApiTags } from "@nestjs/swagger";
 import { JwtAuthGuard } from "../auth/jwt-auth.guard";
 
 @Controller("users")
 @ApiTags("users")
 export class UsersController {
     constructor(private readonly usersService: UsersService) {}
 
+    @ApiOperation({
+        summary: "Gets all users.",
+    })
     @Get()
     findAll() {
         return this.usersService.findAll();
     }
 
+    @ApiOperation({
+        summary: "Gets a unique user on login.",
+    })
     @UseGuards(JwtAuthGuard)
     @ApiBearerAuth()
     @Get("me")
     getProfile(@Request() req) {
         return this.usersService.getPrivateUserProfile(req.user.userId);
     }
 
+    @ApiOperation({
+        summary: "Gets a user with full details.",
```

Comment:
maybe "Gets a user with full details given a userId (uuid)." like the other ones

Reply:
I updated so the paramater is noted as userId (int) so it's follows the pattern of all @Param inputs and doesn't get confused with the ones where we pull from the jwt token which were noted with uuid.